doc/circuitikzmanual.tex

```diff
@@ -1706,17 +1721,18 @@ \subsection{Electronic Tubes}
     \circuitdesc*{magnetron}{Magnetron}{}( anode/-90/0.2, cathode1/135/0.2,
     cathode2/45/0.2, left/180/0.2, right/0/0.2, top/90/0.4 )
 \end{groupdesc}
+
 \begin{LTXexample}[varwidth=true]
-	\begin{circuitikz}
-	\draw (0,-2)node[rground](gnd){} to[voltage source,v<={HV}]++(0,3)--++(1,0)to[V,n=DC]++(2,0);
-	\draw (2,-1) node[magnetron,scale=1](magn){};
-	\draw (DC.left)++(-0.2,0)to [short,*-] ++(0,-1) to [short] (magn.cathode1);
-	\draw (DC.right)++(0.2,0)to [short,*-] ++(0,-1) to [short] (magn.cathode2);
-	\draw (magn.anode) to [short] (magn.anode|-gnd) node[rground]{};
-	\draw (magn.cathode1)node[above]{$1$};
-	\draw (magn.cathode2)node[above]{$2$};
-	\draw[->](magn.east) --++(1,0)node[right]{$RF_{out}$};
-	\end{circuitikz}
+\begin{circuitikz}
+\draw (0,-2)node[rground](gnd){} to[voltage source,v<={HV}]++(0,3)--++(1,0)to[V,n=DC]++(2,0);
+\draw (2,-1) node[magnetron,scale=1](magn){};
+\draw (DC.left)++(-0.2,0)to [short,*-] ++(0,-1) to [short] (magn.cathode1);
+\draw (DC.right)++(0.2,0)to [short,*-] ++(0,-1) to [short] (magn.cathode2);
```

Comment:
I think I've messed up the indentation on these, and I can't seem to restore it.  Other than that, I think that this is in a mergable state.